vue/src/Debugging/Debugging.vue

```diff
@@ -170,7 +170,8 @@
               <tbody>
               <tr>
                 <td style="word-break: break-all">
-                  {{ selectedEvent?.eventData && JSON.stringify(selectedEvent.eventData) }}
+                  {{ selectedEvent?.eventData &&
+                  JSON.stringify(selectedEvent.eventData, getCircularReplacer) }}
```

Comment:
@diosmosis It works for non angular sites, but it doesn't work for angular sites

Reply:
I have shared the details on slack on how to reproduce it.

Reply:
@diosmosis I tried using `{{ selectedEvent?.eventData }}` the error goes away but it prints incorrectly
![Screenshot from 2022-06-03 10-25-44](https://user-images.githubusercontent.com/14902277/171794396-014ba12d-f92c-4898-93aa-581df043da64.png)

we need to stringify the JSON and also to avoid circular dependency we need the same, the issue is not because of angular site only, its evident in such sites due to `__ngContext__` having such circular dependency

Reply:
Ok. In that case I would say the code's fine except instead of calling `JSON.stringify(..., getCircularReplacer())` in the template on every update, it should be in computed properties, eg:

```
computed: {
  selectedEventData() {
    return this.selectedEvent?.eventData && JSON.stringify(this.selectedEvent.eventData, getCircularReplacer());
  },
},
```

then in the template:

```
{{ selectedEventData }}
```

The code looks fine otherwise, though I didn't test it locally.

Reply:
@diosmosis updated as suggested